### DISCUSSION TITLE
Milvus

### DISCUSSION THREAD

**Discussion by Boganabi** [DrAlzahraniProjects/csusb_fall2024_cse6550_team1#17] [General]
Discuss things related to Milvus here!

**Comment by itsnotmik** [comment]
Milvus documentation and installation available here: https://milvus.io/docs/install_standalone-docker.md

**Comment by ACraig7** [comment]
Milvus is [open source](https://milvus.io/docs/product_faq.md#How-much-does-Milvus-cost).
